portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,84 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body {
+  background-color: #aaa;
+  margin: 0;
+  padding: 0;
+}
+
+#header-content {
+  background-color: #aaa;
+  display: flex;
+  height: 5%;
+  margin: 0;
+  padding: 0;
+  width: inherit;
+}
+
+#image-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#cycle-button {
+  background-color: #888;
+  color: #000;
+  border: none;
+}
+
+#image-container button:hover {
+  background-color: #00f;
+}
+
+#header-selfie {
+  height:450px; 
+  width:300px;
 }
 
 #greeting-container {
-  margin-top: 20px;
+  background-image: url(images/skyline.jpg);
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+#greeting-container h1,
+ h3 {
+  color: #F00;
```

Comment:
```css
  color: #f00;
```
Only user lowercase for property values. See https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
Done.